[2020-04-26 19:33:29,402] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T19:33:23.505550+00:00 [queued]>
[2020-04-26 19:33:29,488] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T19:33:23.505550+00:00 [queued]>
[2020-04-26 19:33:29,492] {{taskinstance.py:866}} INFO - 
--------------------------------------------------------------------------------
[2020-04-26 19:33:29,493] {{taskinstance.py:867}} INFO - Starting attempt 1 of 2
[2020-04-26 19:33:29,494] {{taskinstance.py:868}} INFO - 
--------------------------------------------------------------------------------
[2020-04-26 19:33:29,647] {{taskinstance.py:887}} INFO - Executing <Task(PythonOperator): persiste_dados_mongodb> on 2020-04-26T19:33:23.505550+00:00
[2020-04-26 19:33:29,665] {{standard_task_runner.py:53}} INFO - Started process 7075 to run task
[2020-04-26 19:33:29,830] {{logging_mixin.py:112}} INFO - Running %s on host %s <TaskInstance: ReclameAqui.persiste_dados_mongodb 2020-04-26T19:33:23.505550+00:00 [running]> 030a07de9129.semantix
[2020-04-26 19:33:29,869] {{logging_mixin.py:112}} INFO - DEBUG: {'company': 'Cielo', 'date': '20200426'}
[2020-04-26 19:33:29,881] {{logging_mixin.py:112}} INFO - 200
[2020-04-26 19:33:29,882] {{logging_mixin.py:112}} INFO - {"id": "20200426CIELO"}
[2020-04-26 19:33:29,882] {{python_operator.py:114}} INFO - Done. Returned value was: None
[2020-04-26 19:33:29,892] {{taskinstance.py:1048}} INFO - Marking task as SUCCESS.dag_id=ReclameAqui, task_id=persiste_dados_mongodb, execution_date=20200426T193323, start_date=20200426T193329, end_date=20200426T193329
[2020-04-26 19:33:39,315] {{logging_mixin.py:112}} INFO - [2020-04-26 19:33:39,314] {{local_task_job.py:103}} INFO - Task exited with return code 0
